src/lib/rnp2.c

```diff
@@ -897,7 +902,7 @@ rnp_generate_private_key(rnp_keyring_t keyring,
 }
 
 rnp_result_t
-rnp_key_free(rnp_key_t *key)
+rnp_key_free(rnp_key_t key)
```

Comment:
If we have `rnp_key_free(rnp_key_t *key)` then we can NULL the pointer out on the caller's side with `*key = NULL` after freeing. Do you feel it's not really worth it?

Reply:
Fine either way, I guess nulling it out will probably save someone a crash someday though.

Reply:
It does seem like the chances of it being beneficial are pretty low and it makes the API slightly more difficult to use, so your way is probably better.